dali/test/python/test_operator_color_twist.py

```diff
@@ -0,0 +1,39 @@
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import numpy as np
+import os
+
+from test_utils import compare_pipelines
+from test_utils import RandomDataIterator
+from test_utils import RandomDataIterator
+
+class ColorTwistPipeline(Pipeline):
+    def __init__(self, batch_size, seed, data_iterator, old=False, num_threads=1, device_id=0):
+        super(ColorTwistPipeline, self).__init__(batch_size, num_threads, device_id, seed=seed)
+        self.iterator = data_iterator
+        self.input = ops.ExternalSource()
+        self.hue = ops.Uniform(range=[-20., 20.])
+        self.sat = ops.Uniform(range=[0., 1.])
+        self.bri = ops.Uniform(range=[0., 2.])
+        self.con = ops.Uniform(range=[0., 2.])
+        self.color_twist = ops.OldColorTwist(device="gpu") if old else ops.ColorTwist(device="gpu") 
+
+
+    def define_graph(self):
+        self.images = self.input()
+        return self.color_twist(self.images.gpu(), hue=self.hue(), saturation=self.sat(), brightness=self.bri(), contrast=self.con())
+    
+    def iter_setup(self):
+        self.feed_input(self.images, self.iterator.next(), layout="HWC")
```

Comment:
Please use the new ExternalSource API and forget about iter_setup.

Reply:
done